resources/certificate.rb

```diff
@@ -31,7 +31,15 @@
 
   java_home = new_resource.java_home
   keytool = "#{java_home}/bin/keytool"
-  truststore = new_resource.keystore_path
+  truststore = if new_resource.keystore_path.empty?
+                 if ::File.exist?("#{java_home}/lib/security/cacerts")
```

Comment:
testing if a file is present is not the best test since someone could touch a file and this would break, we could either key off `node['java']['install_type']` or make that a property to pass in.